features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/APIClient.js

```diff
@@ -16,7 +16,8 @@
  * under the License.
  */
 "use strict";
-import SwaggerClient from 'swagger-client'
+//TODO: [rnk] Change to {from 'swagger-client'} after swaggerjs resolved the issue
+import SwaggerClient from '../../../../public/js/swagger-client'
```

Comment:
Shall we do what @pahans suggested @code review, Push the patched swagger-client library to out pvt github repo and refer that repo in application `package.json` file

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/4809/commits/d392a1174576ade13b8a8b4e986b7f808d5c5253